app/models/publisher.rb

```diff
@@ -217,6 +217,10 @@ def history
     combined.map { |c| c.values.first }
   end
 
+  def deleted?
+    last_status_update.present? && last_status_update.status == PublisherStatusUpdate::DELETED
```

Comment:
I believe this will result in 2 db calls, probably better to use the following 
```suggestion
    last_status_update&.status == PublisherStatusUpdate::DELETED
```